src/app/shared/components/template/components/progress-path/progress-path.component.html

```diff
@@ -0,0 +1,16 @@
+<div class="progress-path-wrapper">
+  @for (childRow of _row.rows | filterDisplayComponent; track trackByRow($index, childRow)) {
+    <div
+      class="progress-path-child-wrapper"
+      [class]="{ odd: $index % 2 === 0, even: $index % 2 !== 0 }"
```

Comment:
nit(non-blocking): interesting, I thought you needed to use `ngClass` to support this type of syntax, nice it works inline! Although I'm guessing you only actually need to track the even state, and actually angular provides `$even` and `$odd`  context variables
https://angular.dev/api/core/@for